nucliadb_writer/nucliadb_writer/api/v1/upload.py

```diff
@@ -537,13 +538,15 @@ async def generate_buffer(storage_manager, request):
     except LimitsExceededError as exc:
         raise HTTPException(status_code=402, detail=str(exc))
 
-    headers = {}
-    headers["NDB-Seq"] = f"{seqid}"
-    headers["NDB-Resource"] = f"/{KB_PREFIX}/{kbid}/resources/{rid}"
-    headers["NDB-Field"] = f"/{KB_PREFIX}/{kbid}/resources/{rid}/field/{valid_field}"
+    response.headers["NDB-Seq"] = f"{seqid}"
+    response.headers["NDB-Resource"] = f"/{KB_PREFIX}/{kbid}/resources/{rid}"
```

Comment:
What about changing `resources` for the `RESOURCE_PREFIX` constant instead?

Reply:
This headers will be removed, I'm uploading a new version